python/svix/api/common.py

```diff
@@ -27,6 +27,21 @@ def sanitize_field(v: t.Any) -> t.Any:
     return v
 
 
+def _serialize_single_param(val: t.Any) -> str:
+    if isinstance(val, datetime):
+        if val.tzinfo is None:
+            val.replace(tzinfo=timezone.utc)
+        return val.isoformat()
+    elif isinstance(val, bool):
+        return str(val).lower()
```

Comment:
```suggestion
        return "true" if val else "false"
```
Be explicit with what you want. You don't want a lower case string representation of boolean, you want "true" if val, and "false" otherwise.